### ISSUE TITLE
The details of pretraining ARC

### ISSUE THREAD

**Issue by StarBlue98** [LeapLabTHU/ARC#15]
Thanks for your impressive work! 
I wonder that how many epochs are needed to train Imagenet and then use the pretrain model to train the DOTA dataset? And is the results in Table. 3  under single-scale training ?

**Comment by yifanpu001** [comment]
Hi, we pretrain the backbone on ImageNet for 100 epoches with cosine learning rate. Results in Tab.3 are all under single-scale training and inference
